client/.npmrc

```diff
@@ -0,0 +1,2 @@
+# TODO: turn this off at some point
+legacy-peer-deps = true
```

Comment:
What does this do?

Reply:
NPM has "peer dependencies", or dependencies that are... for lack of a better word, tangential? The main purpose of them is for packages that are modules or extensions to existing ones, like a plugin for Webpack. Webpack would be the peer dependency in that instance.

Recent NPM versions began to automatically resolve and install peer dependencies, which causes issues when the dependencies have incompatible versions (not necessarily actually incompatible). I think the whole ecosystem is still adjusting to it, and I was getting some install errors and I didn't want CI or something to fail.